Installation.bat

```diff
@@ -0,0 +1,17 @@
+haxelib install lime

```

Comment:
Where did the start line go? Where is the function to start this?

Reply:
> Where did the start line go? Where is the function to start this?

It opens the terminal automatically when you press on it and puts it in. You have to read README.md for the steps on using it and shit.